CHANGELOG.rst

```diff
@@ -2,14 +2,192 @@
+
+Detailed changes
+----------------
+
+The 2.6.0-alpha.0 changes are included as well, see the earlier changelog entry.
+
+**New features**
+
+* [#3688] Objects API registration rework
+
+    - Added support for selecting an available object type/version in a dropdown instead
+      of copy-pasting a URL.
+    - The objecttype definition (JSON-schema) is processed and will be used for validation.
+    - Registration configuration is specified on the "variables" tab for each available
+      (built-in or user-defined) variable, where you can select the appropriate object
+      type property in a dropdown.
+    - Added the ability to explicitly map a file upload variable into a specific object
+      property for better data quality.
+    - Ensured that the legacy format is still available (100% backwards compatible).
+
+* [#3855] Improved user experience of DMN integration
+
+    - The available input/output parameters can now be selected in a dropdown instead of
+      entering them manually.
+    - Added robustness in case the DMN engine is not available.
+    - Added caching of DMN evaluation results.
+    - Automatically select the only option if there's only one.
+
+* Added documentation on how to configure Camunda for DMN.
+* Tweaked the dark-mode styling of WYSIWYG editors to better fit in the page.
+* [#3164] Added explicit timeout fields to services so they can be different from the
+  global default.
+* [#3695] Improved login screen and flow
+
+    - Allow opt-in to automatically redirect to OIDC provider.
+    - Support WebAuthn (like YubiKey) hardware tokens.
+
+* [#3885] The admin form list now keeps track of open/collapsed form categories.
+* [#3957] Updated the eIDAS logo.
+* [#3825] Added a well-performing public API endpoint to list available forms, returning
+  only minimal information.
+* [#3825] Added public API endpoint to list available form categories.
+* [#3879] Added documentation on how to add services for the service fetch feature.
+* [#3823] Added more extensive documentation for template filters, field regex validation
+  and integrated this documentation more into the form builder.
+* [#3950] Added additional values to the eHerkenning CSP-header configuration.
+* [#3977] Added additional validation checks on submission completion of the configured
+  formio components in form steps.
+* [#4000] Deleted the 'save and add another' button in the form designer to maintain safe
+  blood pressure levels for users who accidentally clicked it.
+
+**Bugfixes**
+
+* [#3672] Fixed the handling of object/array variable types in service fetch configuration.
+* [#3890] Fixed visually hidden fields not being sent to Objects API registration backend.
+* [#1052] Upgraded DigiD/eHerkenning library.
+* [#3924] Fixed updating of payment status when the "registration after payment is
+  received" option is enabled.
+* [#3909] Fixed a crash in the form designer when removing form variables that are used
+  in the case properties mapping.
```

Comment:
This one is not immediately clear to me

Reply:
it's about zaakeigenschappen, reworded it!